index.js

```diff
@@ -1485,6 +1484,40 @@ Renderer.prototype.draw = function() {
           let cType
           cType = ent.getComponent('Camera')
           if (cType.projection == 'orthographic') {
+            draw = true 
+            console.log(cType)
+            let orthoTransform = ent.getComponent('Transform')
+
+            const dx = (cType.right - cType.left) / (2 / cType.zoom)
```

Comment:
In the orthographic camera entity we do a bunch of maths with near/far/zoom and use it to update the orthographic camera matrix. 
https://github.com/pex-gl/pex-renderer/blob/89bc9d3cac2f559c3706435db8941ad4d3f35d4d/camera.js#L183

This maths has to be redone every frame to calculate the view cube to draw helpers, is it worth saving these values as an object property to save on computation/gc of all these declared variables